bin/ember-component-template-colocation-migrator

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env node
+'use strict';
+
+// Provide a title to the process in `ps`
+process.title = 'ember-component-template-colocation-migrator';
+
+let fs = require('fs');
+let Migrator = require('../lib/migrator');
+let nopt = require("nopt");
+let options = {
+  'project-root': [String]
+};
+
+let parsed = nopt(options);
+let projectRoot = parsed['project-root'] || process.cwd();
+
+let migrator = new Migrator({ projectRoot });
+
+migrator.execute().then(function() {
+  console.log('Codemod finished successfully!');
```

Comment:
Not a biggie but an improvement here would be to use something like [`chalk`](https://www.npmjs.com/package/chalk)? That way we can have different colored logs to help readers parse the log output better. 

```js
console.log(chalk.red('This is an error'))
console.log(chalk.green('This is a success'))
console.log(chalk.white('This is information'))
```

Reply:
Seems like a nice improvement, I'm not sure it's worth the effort right now though